common/src/main/kotlin/ImageFormat.kt

```diff
@@ -0,0 +1,16 @@
+package dev.kord.common
+
+enum class ImageFormat(val extension: String) {
+    JPEG("jepg"),
+    JPG("jpg"),
+    GIF("gif"),
+    WebP("webp"),
+    PNG("png"),
+    Lottie("json");
+
+    companion object {
+        fun isSupported(fileName: String): Boolean {
+            return ImageFormat.values().any { fileName.endsWith(it.extension, true) }
+        }
+    }
+}
```

Comment:
We already have something like `ImageFormat`, `Image.Format`. Probably better to use that.